Example/Tests/ContentDynamicFlowLayoutTest.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  ContentDynamicFlowLayoutTest.swift
+//  collection_flow_layout_Example
+//
+//  Created by sergey on 2/28/18.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Quick
+import Nimble
+import collection_flow_layout
```

Comment:
Use @testable import collection_flow_layout and remove all non tests sources from `Compile Sources` phase